backends/client.go

```diff
@@ -15,5 +15,7 @@ func New(config *Config) (Client, error) {
 		return NewDiskBackend(*config)
 	case ConfigTypeMock:
 		return NewMockBackend(), nil
+	case ConfigTypeS3:
```

Comment:
This should move two lines up (in front of `ConfigTypeMock`. `ConfigTypeMock` should always be the last case